### PR TITLE
Remove old detection of propers

### PR DESCRIPTION
**Pull request by fernandog**
@pymedusa/developers 

History.mako is also working

```
2016-07-25 12:43:36 DEBUG    SNATCHQUEUE-MANUALSNATCH-95021 :: [42fd472] Found proper tags for The.Middle.S04E08.PROPER.HDTV.x264-2HD.mp4. Snatching as PROPER
2016-07-25 12:43:36 INFO     SNATCHQUEUE-MANUALSNATCH-95021 :: [42fd472] Downloading The.Middle.S04E08.PROPER.HDTV.x264-2HD.mp4 with 1 seeders and 0 leechers from TorrentProject
2016-07-25 12:43:36 INFO     SNATCHQUEUE-MANUALSNATCH-95021 :: [42fd472] Beginning to manual snatch release: The.Middle.S04E08.PROPER.HDTV.x264-2HD.mp4
```

### CODE REVIEW

**Comment by ratoaq2** [comment]
Should the same be applied here https://github.com/pymedusa/SickRage/blob/92d3153d1a8bf800a3185cbab7548587d9fa269b/sickbeard/search.py#L291 ?

**Comment by duramato** [comment]
We should add something like there is on cache, a proper_tags that way you don't need to have checks/guessit or something else, just call guessit once and store it and its more accurate.
IMO

**Comment by fernandog** [comment]
@duramato but don't you agree the when we snatch a proper status is SNATCHED_PROPER, so we don't need to call anything else.

Also history algorithm does a lof of "groupings".

File: sickbeard/name_parser/parser.py [hunk 1]
```diff
@@ -660,7 +660,7 @@ def is_anime(self):
 
     @property
     def is_proper(self):
-        return re.search(r'(^|[\. _-])(proper|repack)([\. _-]|$)', self.extra_info, re.I) is not None if self.extra_info else False
+        return True if self.proper_tags else False
```
Comment (fernandog) [inline sickbeard/name_parser/parser.py]: @ratoaq2 should we do this?

**Comment by fernandog** [comment]
```
    /home/travis/build/pymedusa/SickRage/sickbeard/postProcessor.py line 1111 in process
      if cur_ep.proper_tags:
   AttributeError: 'TVEpisode' object has no attribute 'proper_tags'
```

File: sickbeard/name_parser/parser.py [hunk 2]
```diff
@@ -658,11 +658,6 @@ def is_anime(self):
             return True
         return False
 
-    @property
-    def is_proper(self):
-        # Part of TVEpisode object and it's inserted in 'tv_episodes' table, 'is_proper' column
```
Comment (fernandog) [inline sickbeard/name_parser/parser.py]: @medariox so from where is_proper will get its value ?

as I said in IRC:
```
    /home/travis/build/pymedusa/SickRage/sickbeard/postProcessor.py line 1111 in process
      if cur_ep.proper_tags:
   AttributeError: 'TVEpisode' object has no attribute 'proper_tags'
```

TVEpisode is created using tv_episodes table
and to populate column "is_proper" that line in parser is needed

**Pull request merged**